.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+    env: {
+        browser: true,
+        es2021: true
+    },
+    extends: ['airbnb-base', 'prettier'],
+    parserOptions: {
+        ecmaVersion: 12,
+        sourceType: 'module'
+    },
+    rules: {
+        'prettier/prettier': 'error',
+    },
+    plugins: ['prettier'],
+};
```

Comment:
```suggestion
{
  "extends": ["airbnb-base", "prettier"],
  "plugins": ["prettier"],
  "rules": {
    "prettier/prettier": ["error"]
  }
}
```

Let's rename this file to `.eslintrc.json`

Reply:
Reference: https://www.robinwieruch.de/prettier-eslint